.github/workflows/phase-1.yml

```
@@ -13,123 +21,145 @@ jobs:
+
+    # 1-2. ステージング向けビルドブランチで、Bundle.jsを生成してコミットする
+    - name: STG checkout
       uses: actions/checkout@v2
       with:
-        ref: develop
+        ref: ${{ env.STAGING_BUILD_BRANCH }}
+
 
-    - name: Create PR ( develop -> release_YYMMDD )
+    - name: STG yarn webpack:staging
+      run: |
+        # TODO: staging環境ができたら、yarn webpack:stagingに変更すること
+        yarn
+        yarn webpack:demo
+
+    - name: STG check diff
+      id: staging_diff
+      run: |
+        staging_diff=$(git diff)
+        echo "::set-output name=stdout_staging::$staging_diff"
+
+    - name: STG git push build file
+      if: ${{ steps.staging_diff.outputs.stdout_staging }}
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git add .
+        git commit -m "add: staging build dist $DATE" --no-verify
+        git push
+
+
+    # 1-3. stagingにPRを出す
+    - name: STG Create PR
+      if: ${{ steps.staging_diff.outputs.stdout_staging }}
       uses: actions/github-script@v4
       with:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         script: |
           github.pulls.create({
             owner: context.repo.owner,
             repo: context.repo.repo,
-            title: `develop -> ${process.env.RELEASE_BRANCH_NAME}`,
-            head: "develop",
-            base: `${process.env.RELEASE_BRANCH_NAME}`
+            title: `${process.env.STAGING_BUILD_BRANCH} -> demo`,
+            head: `${process.env.STAGING_BUILD_BRANCH}`,
+            base: "demo"
           })
 
-    # 作成したPRのNumberを取得
-    - name: Get PR Number
-      id: get-pr-number
-      uses: actions/github-script@v4
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        result-encoding: string
-        script: |
-          const prList = await github.pulls.list({
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            sort: "created",
-            base: `${process.env.RELEASE_BRANCH_NAME}`
-          })
-          console.log(prList)
-          console.log(prList.data[0].number)
-          return prList.data[0].number
 
-    - name: Merge PR
-      uses: actions/github-script@v4
+
+    # 2. master向け
+    # 2-1. master向けビルドブランチをdevelopから切り出す
+    - name: checkout develop
+      uses: actions/checkout@v2
       with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        script: |
-          github.pulls.merge({
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            pull_number: ${{ steps.get-pr-number.outputs.result }}
-          });
+        ref: develop
+
+    - name: PRD Create build branch
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        git checkout -b "$PRODUCTION_BUILD_BRANCH"
+        git push origin "$PRODUCTION_BUILD_BRANCH"
 
-    # 3. release_YYMMDD で build.js をビルドする
-    - name: checkout release_YYMMDD
+    # 2-2. ステージング向けビルドブランチで、Bundle.jsを生成してコミットする
```

```suggestion
    # 2-2. master向けビルドブランチで、Bundle.jsを生成してコミットする
```
`[nits]` これ文言は間違いで良いですよね？良ければサジェスチョンコミットお願いします！